bubblewrap.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: bubblewrap
+  version: 0.6.2
+  epoch: 0
+  description: "Unprivileged sandboxing tool"
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+      - "*"
+      attestation: TODO
+      license: LGPL-2.0-or-later
+  dependencies:
+    runtime:
+
+environment:
+  contents:
+    keyring:
+      - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+    repositories:
+      - https://packages.wolfi.dev/os
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - build-base
+      - bash
+      - meson
+      - libcap-dev
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/containers/bubblewrap/archive/refs/tags/v${{package.version}}.tar.gz
+      expected-sha256: f8d21707b492d318380f0a76966e8cfa40c2df283de8bcf1ac51e7a4dcc12e97
+  - runs: |
+      meson \
+        --prefix=/usr \
+        --sysconfdir=/etc \
+        --mandir=/usr/share/man \
+        --localstatedir=/var \
+        --buildtype=plain \
+        -Drequire_userns=true . output
+      meson compile -C output
+      DESTDIR="${{targets.destdir}}" meson install --no-rebuild -C output
```

Comment:
Should have docs subpackage, right?  It generates a manpage I think.

Reply:
no, no manpages:
```
$ tar -xvf packages/x86_64/bubblewrap-0.6.2-r0.apk
x .PKGINFO
x usr
x usr/bin
x usr/bin/bwrap
x usr/share
x usr/share/bash-completion
x usr/share/bash-completion/completions
x usr/share/bash-completion/completions/bwrap
x usr/share/zsh
x usr/share/zsh/site-functions
x usr/share/zsh/site-functions/_bwrap
```

Reply:
the meson command here is gleaned from the alpine build which uses `abuild-meson`, which appears to do just the following:
https://gitlab.alpinelinux.org/alpine/abuild/-/merge_requests/60/diffs

Reply:
maybe since we dont have the `docbook-xsl` dependency?

Reply:
https://git.alpinelinux.org/aports/tree/main/bubblewrap/APKBUILD#n11

Reply:
oh, right, i didn't package any of that stuff because it was annoying.